sysutils/neovim-remote/Portfile

```
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                neovim-remote
+version             2.4.0
+revision            0
+
+categories          sysutils editors
+platforms           darwin
+supported_archs     noarch
+license             MIT
+maintainers         {@gpanders gpanders.com:greg} openmaintainer
+
+description         Control nvim processes using "nvr" commandline tool
+long_description \
+    This package provides an executable called nvr which allows you to \
+    control nvim processes from the shell and open files within :terminal \
+    without starting a nested nvim process.
+
+homepage            https://github.com/mhinz/${name}
+
+checksums           rmd160  2a279a1690b9b8949a14bd795bc14c7bd2bf28e3 \
+                    sha256  f199ebb61c3decf462feed4e7d467094ed38d8afaf43620736b5983a12fe2427 \
+                    size    15315
+
+python.default_version \
+                    39
+
+depends_build-append \
+                    port:py${python.version}-setuptools
+
+depends_lib-append \
+                    port:py${python.version}-neovim \
+                    port:py${python.version}-psutil
```

```suggestion
depends_run-append \
                    port:py${python.version}-neovim \
                    port:py${python.version}-psutil
```

I don't think these will be required at build time, and so should instead just be a runtime dependency.